modules/microshift-oc-mirror-embed-ops-disconnected-use.adoc

```diff
@@ -4,24 +4,25 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="microshift-oc-mirror-prep-ops-cat-images-disconnected-use_{context}"]
-= Getting catalogs and Operator container image references to use with RHEL for Edge in disconnected environments
-
-After performing a dry run with the oc-mirror plugin to review the list of images that you want to mirror, you must get all of the container image references, then format the output for adding to an Image Builder blueprint.
+= Getting catalogs and Operator container image references
 
+After performing a dry run with the oc-mirror plugin to review the list of images that you want to mirror, you must get all of the container image references, then format the output for adding to an image builder blueprint.
+//does oc mirror v2 still use a dry run? this is a different procedure from the one in "Creating custom catalogs using the oc-mirror plugin"
```

Comment:
--dry-run is used to see what images it would mirror with out performing the actual mirroring. yes oc mirror v2 uses dry-run. Did not really understand your question about `creating custom catalogs using the oc-mirror plugin`